CMakeLists.txt

```
@@ -362,6 +362,15 @@ if(NOT Launcher_FORCE_BUNDLED_LIBS)
 
     # Find qrcodegencpp-cmake
     find_package(qrcodegencpp QUIET)
+
+    find_package(LibArchive 3.7.8 QUIET)
+    # Fallback to pkg-config (if available) if CMake files aren't found
+    if(NOT LibArchive_FOUND)
+        find_package(PkgConfig)
+        if(PkgConfig_FOUND)
+            pkg_check_modules(LibArchive IMPORTED_TARGET LibArchive>=3.7.8)
+        endif()
+    endif()
```

```suggestion
    find_package(PkgConfig REQUIRED)
    pkg_check_modules(libarchive REQUIRED IMPORTED_TARGET libarchive)
```
I don't know of any distribution of libarchive that has CMake support...